libs/Spectra.py

```diff
@@ -23,3 +29,14 @@ def save_to_msp(self, filename):
         :param filename: target MSP file
         """
         save_as_msp(self.spectrums, filename)
+
+
+def spectra_eq(first, second):
+    """
+    Compare two Spectra objects.
+    Native __eq__ definition does not work properly.
+
+    :param first: spectra object
+    :param second: spectra object
+    """
+    return first.peaks == second.peaks and first.losses == second.losses and first.metadata == second.metadata
```

Comment:
What is the `.losses` attribute?

Reply:
I'm not sure, I was trying to replicate behavior of `matchms/`[`Spectrum.__eq__`](https://github.com/matchms/matchms/blob/d5c6cd79c891325815099c87bf6dfe2ff2db562b/matchms/Spectrum.py#L75) with only change in `metadata` treatment.